src/engine/Resources/Sound.ts

```diff
@@ -274,8 +276,15 @@ export class Sound implements ILoadable, IAudio {
 
    /**
     * Play the sound, returns a promise that resolves when the sound is done playing
+    * Valid string inputs include:
+    * -'linearUp'
+    * -'linearDown'
+    * -'ease'
+    * -'ease-in'
+    * -'ease-out'
+    * -'ease-in-out'
     */
-   public play(volume?: string): Promise<boolean> {
+   public play(data?: string | number | IBezierPoints): Promise<boolean> {
```

Comment:
In this case, interface will just be an object, so you can determine if it's a string it's an easing constant or if it's an object, it's (probably) the interface.

```ts
if (typeof arg === 'string') {
// easing string
} else if (typeof arg === 'object') {
// IBezierPoints implementation
}
```

If you wanted to strongly type the IBezierPoints check, you can use [user-defined type guards](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards).

```ts
function isBezierPoints(arg: object): arg is IBezierPoints {
   return (<IBezierPoints>arg).point1;
}

Does that help?